packages/fuel-indexer-lib/src/graphql/parser.rs

```diff
@@ -144,6 +144,10 @@ pub struct ParsedGraphQLSchema {
     /// All unique names of types in the schema (whether objects, enums, or scalars).
     type_names: HashSet<String>,
 
+    /// Mapping of lowercase entity names to their actual entity types.
+    // Used to refer to top-level entities in GraphQL queries.
+    entity_names_to_types: HashMap<String, String>,
```

Comment:
- So going forward let's be a bit better/consistent about the naming. 
- If these are `TypeDefinition` names (looks like they are?) then let's refer to them as that instead of "entity"